### PR TITLE
Hook up to CI

### PR DESCRIPTION
**Pull request by barrettk**

### CODE REVIEW

**Comment by barrettk** [comment]
Think we have to remove the check part to figure out the tests first. If we remove both all we're really doing is installing the packages, so IMO we should get at least one of them to work

File: README.Rmd [hunk 1]
```diff
@@ -13,4 +13,54 @@ knitr::opts_chunk$set(
...
+You can install the latest development version from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("metrumresearchgroup/pmforest", ref = "develop")
+```
+
+## Documentation
+
+You can find documentation and a "Getting Started" vignette that shows users how to set up and summarize their data, as well as demonstrates the basic plotting methods. Additional formatting attributes are highlighted as well.
```
Comment (seth127) [inline README.Rmd:40]: missing "here" (with link) from the end of that first sentence. Should be 
```
You can find documentation and a "Getting Started" vignette that shows users how to set up and summarize their data, as well as demonstrates the basic plotting methods, [here](https://metrumresearchgroup.github.io/pmforest/).
```
```

File: README.Rmd [hunk 2]
```diff
@@ -13,4 +13,54 @@ knitr::opts_chunk$set(
...
+
+
+### Featured Vignettes
+
+* [Getting Started with bbr](https://metrumresearchgroup.github.io/pmforest/articles/getting-started.html) -- Data specifications, summarization, and basic plotting methods.
```
Comment (seth127) [inline README.Rmd:45]: Oops, change to "Getting Started with pmforest"

**Review by seth127 — APPROVED** [review]
Everything is running CI. I rendered the pkgdown locally and it looks great. Also pushed a tag for a dev release and it appears to have worked: https://github-drone.metrumrg.com/metrumresearchgroup/pmforest/15

Let' merge this , and then test pulling that `0.0.0.9000` from s3 via pkgr and try it out to be sure it worked, but then I think we're ready for `0.0.1`!

**Pull request merged**